components/scream/ekat/src/ekat/util/scream_test_utils.hpp

```diff
@@ -73,6 +72,8 @@ void catch2_req_pk_sensitive(const Scalar lhs, const Scalar rhs)
   }
 }
 
+int get_test_device (const int mpi_rank);
```

Comment:
This seems like it encapsulates a concept that is now very critical to our testing process. Could you provide some comments about this here?

Reply:
done